doc/deploy.md

```diff
@@ -1,49 +1,48 @@
 
-The current testnet environment lives in [XMTP Sepolia Chain](https://xmtp-testnet.explorer.alchemy.com/).
+The XMTP testnet nodes operated by Ephemera include:
 
-### Deploy XMTPD nodes
-
-Node deployment is currently fully handled by [Ephemera](https://github.com/ephemeraHQ) and only members of `@ephemerahq/backend` have access to it.
-
-The nodes run by Ephemera are:
-
-| DNS Name                           | Location   | Public Key                                                           |
+| DNS name                           | Location   | Public key                                                           |
 | ---------------------------------- | ---------- | -------------------------------------------------------------------- |
 | https://grpc.testnet.xmtp.network  | US-EAST-2  | 0x03e5442c5d1fe2f02b6b9a1a386383a7766860b40a6079a0223994ffa2ce10512c |
 | https://grpc2.testnet.xmtp.network | EU-NORTH-1 | 0x02fc261d43a0153539a4c64c29763cb0e7e377c0eac2910c3d4bedb2235ac70371 |
 
-For more info, refer to the infrastructure README.
-
-## Local developer environment
-
-Refer to [XMTP Contracts](https://github.com/xmtp/smart-contracts) for further information regarding the on-chain protocol, deployments and source code.
+## Deploy a local development environment
 
-There are two ways of deploying a local environment:
+Use one of the following methods to deploy a local development environment:
 
+- Use the XMTP Contracts Image
+- Use the `dev/up` automation
+  
 ### Use the XMTP Contracts Image
 
-The [XMTP Contracts Image](https://github.com/xmtp/smart-contracts/blob/main/doc/xmtp-contracts-image.md#using-the-image) can be used to deploy a local environment and test `xmtpd` with it.
+You can use the XMTP Contracts Image to deploy a local development environment, and test `xmtpd` with it. 
 
-The documentation contains the deterministic addresses where all the contracts are deployed, to ease the setup step.
+The XMTP Contracts Image [documentation](https://github.com/xmtp/smart-contracts/blob/main/doc/xmtp-contracts-image.md#using-the-image) contains the deterministic addresses where all the contracts are deployed, to ease the setup step.
 
-`dev/local.env` contains by default sane values for local deployments. Modify what is necessary.
+By default, `dev/local.env` contains sane values for local deployments. Modify what's necessary.
+
+For more information about the onchain protocol, deployments, and source code, see the XMTP [smart-contracts](https://github.com/xmtp/smart-contracts) repo.
 
 ### Use the dev/up automation
 
-Use the script provided in `dev/up`, which will automatically handle the deployment for you. The blockchain is started at <http://localhost:7545/>
+Use the script provided in `dev/up` to automatically handle the deployment for you. The blockchain starts at <http://localhost:7545/>.
+
+This method automatically pre-registers two nodes.
 
-This method automatically pre-register two nodes.
+## Register a node
```

Comment:
Is this here to tell us how to register an Ephemera-run node, as well as how to register a node for use with a local dev env -- which is not on the testnet?

Reply:
Ideally this would tell the user how to register nodes manually to use the local development network.